docs/react-native-devtools.md

```diff
@@ -0,0 +1,99 @@
+---
+id: react-native-devtools
+title: React Native DevTools
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+React Native DevTools is our new debugging experience featuring an end-to-end rewrite of our debugger stack. It aims to be more deeply integrated and fundamentally more reliable than previous debugging methods in React Native.
+
+![React Native DevTools opened to the "Welcome" pane](/docs/assets/debugging-rndt-welcome.jpg)
+
+React Native DevTools is designed for debugging React app concerns, and not to replace native tools. If you want to inspect React Native’s underlying platform layers (for example, while developing a Native Module), please use the debugging tools available in Android Studio and Xcode (see [Debugging Native Code](http://localhost:3000/docs/next/debugging-native-code)).
+
+<details>
+<summary>**💡 Compatibility** — released in 0.76</summary>
+
+React Native DevTools supports all React Native 0.76 apps (and onwards) running Hermes. It replaces the previous Flipper, Experimental Debugger, and Hermes debugger (Chrome) entry points.
+
+It is not possible to set up React Native DevTools with any older versions of React Native.
+
+- **Chrome Browser DevTools — unsupported**
+  - Connecting to React Native via `chrome://inspect` is no longer supported. Features may not work correctly, as the latest versions of Chrome DevTools (which are built to match the latest browser capabilities and APIs) have not been tested, and this frontend lacks our customisations. Instead, we ship a supported version with React Native DevTools.
+- **Visual Studio Code — unsupported** (pre-existing)
+  - Third party extensions such as [Expo Tools](https://github.com/expo/vscode-expo) and [Radon IDE](https://ide.swmansion.com/) may have improved compatibility, but are not directly supported by the React team.
```

Comment:
This reads in a negative way. Can we just omit them?

Reply:
The third party VS Code extensions part?

I think we need to be accurate and up front about our compatibility changes, here in our docs as well as the feedback thread. This is both to highlight the depth of the changes made to our debugging stack, and to protect from the possible image that RNDT is meant to work perfectly outside our first party frontend (it very much will not!).

Note that these bullets are collapsed in an expandable `<details>` block.